app_utils/system.py

```
@@ -2455,6 +2457,99 @@ def _collect_rtc_status(logger) -> Dict[str, Any]:
+        client = get_redis_client(max_retries=1)
+    except Exception as exc:
+        if logger:
+            logger.debug("hardware_subsystems: redis connect failed: %s", exc)
+        return result
+    if client is None:
+        return result
+
+    summary: Dict[str, Any] = {}
+    try:
+        raw = client.get("hardware:metrics")
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)